### ISSUE TITLE
[Legitimate Site Unblocked] https://cakefoundation.org/

### ISSUE THREAD

**Issue by stripthis** [MetaMask/eth-phishing-detect#12331]
**Domain:**
https://cakefoundation.org/

**Details:** *Some details about the domain (optional)*
CakePHP software foundation website.

- [x] I have checked to make sure that [there is not a duplicate issue](https://github.com/MetaMask/eth-phishing-detect/issues)

**Comment by AlexHerman1** [comment]
nothing is hosted here. we do not unblock domains that are not hosting any content. please open a new PR when you have a site stoodup and can provide additional context/documentation on the nature of the site.
